.github/workflows/daily-testgrid.yaml

```diff
@@ -0,0 +1,43 @@
+name: testGrid-daily
+on:
+  schedule:
+  - cron: "0 0 * * 1-5"
+
+defaults:
+  run:
+    working-directory: ./testgrid/tgrun
+
+jobs:
+  testgrid-init:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: kubeconfig
+      env:
+        KUBE_CONFIG_DATA: ${{ secrets.TESTGRID_KUBE_CONFIG_DATA }}
+      run: |
+        echo "$KUBE_CONFIG_DATA" | base64 --decode > /tmp/config
+
+    - name: setup-go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.15.1
+
+    - name: tgrun-build
+      run: make build
+
+    - name: testgrid-cleanup
+      env:
+        KUBECONFIG: "/tmp/config"
+      run:
+        kubectl delete vmi --all
```

Comment:
This should be on the instance, not in the action. When we run this, we will need to have a dynamic set of more than 1 instance running these. It will be difficult to maintain this list in the action, but instead, the instance can delete them,.

Reply:
Agreed, please see the previous comment on this.